hypervisor.go

```diff
@@ -414,6 +420,11 @@ func getHostMemorySizeKb(memInfoPath string) (uint64, error) {
 
 // RunningOnVMM checks if the system is running inside a VM.
 func RunningOnVMM(cpuInfoPath string) (bool, error) {
+	if runtime.GOARCH == "arm64" {
```

Comment:
Now that we're adding additional architectures, and since the behaviour might be different for each architecture, how about we add an `arch` field to the logs? We could even add this "centrally" by doing it in `SetLogger()` in `api.go`?

Reply:
yep, definitely that's a good idea, let me added it

Reply:
fixed, thanks

Reply:
I did it but @sameo comments
> Right now it's not because nested virt in ARM is not very mature, but I expect at some point we will be able to detect it in an hypervisor agnostic way. So it should not be a qemu specific method if we want e.g. to support other hypervisors and not duplicate that code.

@sameo  @sboeuf  what do you think?

Reply:
@sameo comment makes sense, that's fine to leave it this way then.